packages/next/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -74,6 +76,12 @@ export default class BuildManifestPlugin {
             ? mainJsChunk.files.filter((file: string) => /\.js$/.test(file))
             : []
 
+        const polyfillChunk = chunks.find(
+          c => c.name === CLIENT_STATIC_FILES_RUNTIME_POLYFILLS
+        )
+        const polyfillFiles: string[] = polyfillChunk ? polyfillChunk.files : []
+        // TODO: invariant that polyfill files is only a single file (?)
```

Comment:
We can remove this now that we push all the files to a unique entry (`/_polyfills`).
```suggestion
```